### PR TITLE
Adding Template request file

### PR DESCRIPTION
**Pull request by kocharrahul7**
> Pull Request Template
>
> 

## Description

>Adding in an issue template for Template requests. This is just to allow a user to request a template is created

Fixes # (issue)

## Type of change

> Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

> Please describe the tests that you ran to verify your changes. Provide instructions, so we can reproduce.
> Please also list any relevant details for your test configuration.

- Checked the file on another directory

## Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Templates.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+name: 📓 Template request
+description: Request a template you would like us to make
```
Comment (mohanarpit) [inline .github/ISSUE_TEMPLATE/Templates.yaml:2]: Minor: can we call them app templates? Or something slightly more descriptive. I read the whole text but it wasn't very clear to me that we were talking about app templates till much later.

Comment (Nikhil-Nandagopal) [reply .github/ISSUE_TEMPLATE/Templates.yaml:2]: +1 App template makes sense

File: .github/ISSUE_TEMPLATE/Templates.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  attributes:
+    label: Objective
+    description: Let us know in as much detail what you want your template to do
+  validations:
+    required: true
+
+- type: textarea
+  id: Wireframes
+  attributes:
+    label: Wireframes
```
Comment (Nikhil-Nandagopal) [inline .github/ISSUE_TEMPLATE/Templates.yaml:27]: I think a single line input is enough for title & wireframes

File: .github/ISSUE_TEMPLATE/Templates.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      e.g. Data should be stored in a Firestore database
+  validations:
+    required: true
+    
+- type: textarea
+  attributes:
+    label: Tags
+    description: |
+      List the tags you feel best categorise your template. Include functionality, widgets used, API or databases, industries in which this would be useful, and any other possible tags to help with discovery.
+    value: |
```
Comment (Nikhil-Nandagopal) [inline .github/ISSUE_TEMPLATE/Templates.yaml:47]: We should have separate dropdowns/inputs for 
- Widgets used
- Use case
- Industry
- Data sources used

**Review by Nikhil-Nandagopal — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/Templates.yaml (hunk 2 above)
Comment (kocharrahul7) [reply .github/ISSUE_TEMPLATE/Templates.yaml:27]: Made all the requested changes

**Review by kocharrahul7 — COMMENTED** [review]
Made all the requested changes

File: .github/ISSUE_TEMPLATE/Templates.yaml [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+title: "[Template]: "
+labels: [Template]
+assignees:
+- Kocharrahul8
+body:
+- type: input
+  id: Title
+  attributes:
+    label: Title
+    description: Name your templte, and make sure it is descriptive, so we make what you need
```
Comment (Nikhil-Nandagopal) [inline .github/ISSUE_TEMPLATE/Templates.yaml:12]: @kocharrahul7 there's a typo in template here

**Review by Nikhil-Nandagopal — CHANGES_REQUESTED** [review]

**Pull request merged**